### PR TITLE
Upgrade most LUPS effects to run at 30+ Hz refresh rates 

### PR DESCRIPTION
**Pull request by Beherith**
Look how smooth fusion spheres, shields, and flamethrower distortions are now! All for free too :P

pls test

### CODE REVIEW

**Comment by Beherith** [comment]
@sprunk  might wanna port this

File: lups/particleclasses/jitterparticles2.lua [hunk 1]
```diff
@@ -170,7 +170,7 @@ local time
 
 function JitterParticles2.BeginDrawDistortion()
   glUseShader(billShader)
-  time = Spring.GetGameFrame()*0.01
+  time = (Spring.GetGameFrame() + Spring.GetFrameTimeOffset()) * 0.003333333
```
Comment (sprunk) [inline lups/particleclasses/jitterparticles2.lua:173]:
```suggestion
  time = (Spring.GetGameFrame() + Spring.GetFrameTimeOffset()) * Game.gameFrame / 10 
```

**Comment by sprunk** [comment]
Is it really for free? There are a bunch of added `Spring.GetFrameTimeOffset()` calls.

**Comment by icexuick** [comment]
Do you need my review on this?

**Comment by Beherith** [comment]
> Is it really for free? There are a bunch of added `Spring.GetFrameTimeOffset()` calls.

I lied. Delete your pr to zk as it will tank performance .

**Comment by Beherith** [comment]
Wow super nice stuff by gh:
image

**Pull request merged**